reference-apps/aaos-unbundled/mediacompose/src/main/java/com/android/designcompose/reference/mediacompose/MediaAdapter.kt

```diff
@@ -326,19 +329,30 @@ class MediaAdapter(
             return
         }
         when (newBrowsingState.mConnectionStatus) {
-            MediaBrowserConnector.ConnectionStatus.CONNECTING -> {}
+            MediaBrowserConnector.ConnectionStatus.CONNECTING -> {
+                canSearch.value = false
+            }
+            MediaBrowserConnector.ConnectionStatus.NONEXISTENT,
```

Comment:
Why fall back to CONNECTED when connection is NONEXISTENT?

Reply:
Updated to do nothing in this case.